docker/quickstart/docker-compose.consumers-without-neo4j.quickstart.yml

```diff
@@ -23,14 +23,30 @@ services:
     - 9091:9091
   datahub-mce-consumer:
     container_name: datahub-mce-consumer
-    depends_on:
-    - datahub-gms
     environment:
     - MCE_CONSUMER_ENABLED=true
     - KAFKA_BOOTSTRAP_SERVER=broker:29092
     - KAFKA_SCHEMAREGISTRY_URL=http://schema-registry:8081
-    - DATAHUB_GMS_HOST=datahub-gms
-    - DATAHUB_GMS_PORT=8080
+    - DATAHUB_SERVER_TYPE=${DATAHUB_SERVER_TYPE:-quickstart}
```

Comment:
:/

Reply:
Yeah, idk what that DATAHUB_SERVER_TYPE does maybe tracking/usage metrics.

Reply:
Yes, this is for telemetry